runtime/itests/reference-mappings/src/test/java/io/atlasmap/reference/java_to_java/JavaJavaComplexListTest.java

```diff
@@ -102,7 +102,7 @@ public void testGenerateListMappingBasic() throws Exception {
         AtlasMappingService atlasMappingService = new AtlasMappingService(
                 Arrays.asList("io.atlasmap.v2", "io.atlasmap.java.v2", "io.atlasmap.xml.v2"));
         atlasMappingService.saveMappingAsFile(a,
-                new File("src/test/resources/javaToJava/atlasmapping-complex-list-autodetect-base.xml"));
+                new File("target/atlasmapping-complex-list-autodetect-base.xml"));
```

Comment:
why have these paths changed?

Reply:
1. In the java-generated xml files, the indentation is 4 spaces, which does not align with the new standard - 2 spaces
2. Those are existing xml files. Unit tests should not modify pre-defined testing files under /src/test folder. All new output files should be saved under target folder.
3. If the path is not changed, the new generated xml files with 4-space indentation will break code format validation. That's why the path got changed.

Reply:
The tests should not be overriding source files in the first place. The paths should keep the original output structure but within the target directory. I.e. target/reference-mappings/javaToJava/......xml

Reply:
The tests should not be overriding source files in the first place. The paths should keep the original output structure but within the target directory. I.e. target/reference-mappings/javaToJava/......xml

Reply:
Done. Generating output files within target sub directory. Please review.